.circleci/config.yml

```diff
@@ -56,6 +56,19 @@ jobs:
           name: Danger
           command: DANGER_GITHUB_API_TOKEN="15e52de81a772b174cc5""e1813d0083564c69c325" yarn danger ci
 
+  build_review_app:
+    executor: hokusai/deploy
+    steps:
+      - hokusai/setup-docker
+      - hokusai/install-aws-iam-authenticator
+      - hokusai/configure-hokusai
+      - run:
+          name: Install jq
+          command: curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /usr/local/bin/jq
```

Comment:
We could also consider installing jq within the hokusai Docker image: https://github.com/artsy/hokusai/blob/master/Dockerfile